src/docdb/tree/DocDBStoredProcedureTreeItem.ts

```diff
@@ -18,7 +19,7 @@ export class DocDBStoredProcedureTreeItem implements IAzureTreeItem {
     public readonly contextValue: string = DocDBStoredProcedureTreeItem.contextValue;
     public readonly commandId: string = 'cosmosDB.openStoredProcedure';
 
-    constructor(private _endpoint: string, private _masterKey: string, private _isEmulator: boolean, private _collection: DocDBCollectionTreeItem, public procedure: ProcedureMeta) {
+    constructor(private _endpoint: string, private _masterKey: string, private _isEmulator: boolean, private _collection: DocDBCollectionTreeItem | GraphCollectionTreeItem, public procedure: ProcedureMeta) {
```

Comment:
Offline feedback: The only reason _collection is required is for the DocumentClient [here](https://github.com/Microsoft/vscode-cosmosdb/blob/b7188b00986b63e7965ba247d19006126ee99d90/src/docdb/tree/DocDBStoredProcedureTreeItem.ts#L38). Consider: 
1. An interface type to handle this. 
2. Passing just the client. 
3. Passing merely the getDocumentClient. This feels weird to me because a parameterless function storing state seems obfuscated if I were to debug this later. 

@StephenWeatherford was there anything else I missed?